test/support/utils.h

```diff
@@ -792,14 +795,16 @@ transform_reduce_serial(InputIterator first, InputIterator last, T init, BinaryO
     return init;
 }
 
-static const char*
-done()
+int
+done(bool is_passed = true)
```

Comment:
I guess name of argument should be "is_done" or revert logic  - "is_skipped"

Reply:
Applied